### PR TITLE
fix(web/applications): fix banner not disappearing on refresh (boas-1…

### PR DESCRIPTION
**Pull request by lutamKC**
…073)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/views/applications/case-s51/properties/tab-attachments.component.njk [hunk 1]
```diff
@@ -79,5 +82,8 @@
             ],
             rows: attachmentsRows
     }) }}
+	{% else %}
+		<p>There are no attachments on this item.</p>
+	{% endif %}
```
Comment (lutamKC) [inline apps/web/src/server/views/applications/case-s51/properties/tab-attachments.component.njk:87]: little fix not related to the task

File: apps/web/src/server/applications/case/s51/__tests__/applications-s51.test.js [hunk 2]
```diff
@@ -415,6 +420,40 @@ describe('S51 Advice', () => {
...
+						lastName: 's51 lastname',
+						enquiryMethod: 'email',
+						enquiryDate: '2020-01-01',
+						enquiryDetails: 's51 enquiry details',
+						adviser: 's51 adviser',
+						adviceDate: '2021-01-01',
+						adviceDetails: 's51 advice details'
+					});
+
+					expect(response?.headers?.location).toEqual('../../s51-advice/1/properties');
```
Comment (beejjacobs) [inline apps/web/src/server/applications/case/s51/__tests__/applications-s51.test.js:449]: I don't know if you want to check this explicitly, if not you can add `.redirects(1);` to the request to follow the redirect.

**Review by beejjacobs — APPROVED** [review]
LGTM, one question/suggestion.

File: apps/web/src/server/applications/case/s51/__tests__/applications-s51.test.js (hunk 2 above)
Comment (lutamKC) [reply apps/web/src/server/applications/case/s51/__tests__/applications-s51.test.js:449]: the ID contained in the redirecting URL comes from the processing of the advice. I suppose that checking whether it's correct or not, is an extra-check

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**